src/order/conditionally_complete_lattice.lean

```diff
@@ -557,13 +557,7 @@ by haveI := unique_prop hp; exact supr_unique
 @csupr_pos αᵒᵈ _ _ _ hp
 
 lemma csupr_set {s : set β} {f : β → α} : (⨆ x : s, f x) = Sup (f '' s) :=
```

Comment:
Note that we have `Sup_image'` that assumes just `[has_Sup]`.

Reply:
Would you recommend using that in the proof or just removing `csupr_set`?

Reply:
I would drop this lemma. I you prefer to leave it, then please fix typeclass assumptions (`[has_Sup]`).